src/builddocs.jl

```diff
@@ -99,13 +99,14 @@ end
 addentry!(dict, object, entry) = setindex!(dict, entry, object)
 
 function processcomment(meta, state, file, block)
-    mac, line, _ = block
+    mac, line, expr = block
+    code = expr
 
     source = (linenumber(line), file)
     docs   = Docs{meta.data[:format]}(exec(state, mac))
 
     object = Comment()
-    entry  = Entry{:comment}(meta.modname, source, docs)
+    entry  = Entry{:comment}(meta.modname, source, code, docs)
 
     object, entry
 end
```

Comment:
I'm playing around with some experimental things (may have relevance to https://github.com/MichaelHatherly/Lexicon.jl/issues/71, ie document without having to import Docile), so we might be able to unify some internals.